SUMMARY.md

```diff
@@ -39,7 +39,9 @@
       * [Search Aggregates](datasets/mozetl/search_aggregates/reference.md)
       * [Search Clients Daily](datasets/mozetl/search_clients_daily/reference.md)
 * [Tools](tools/README.adoc)
-  * [Overview of Mozilla's Data Pipeline](/concepts/data_pipeline.md)
+  * [Overview of Mozilla's Data Pipeline](/concepts/pipeline/data_pipeline.md)
```

Comment:
Please add a pointer to the In-depth Data Pipeline article from [Is that it?](https://docs.telemetry.mozilla.org/concepts/data_pipeline.html#is-that-it)